src/test/java/io/stargate/sgv2/jsonapi/api/v1/CreateCollectionBackwardCompatibilityIntegrationTest.java

```diff
@@ -0,0 +1,102 @@
+                  PRE_LEXICAL_RERANK_COLLECTION_NAME)));
+
+      // create indexes for the collection
+      String[] createIndexCqls = {
+        String.format(
+            "CREATE CUSTOM INDEX IF NOT EXISTS %s_array_contains ON \"%s\".\"%s\" (values(array_contains)) USING 'StorageAttachedIndex';",
+            PRE_LEXICAL_RERANK_COLLECTION_NAME, keyspaceName, PRE_LEXICAL_RERANK_COLLECTION_NAME),
+        String.format(
+            "CREATE CUSTOM INDEX IF NOT EXISTS %s_array_size ON \"%s\".\"%s\" (entries(array_size)) USING 'StorageAttachedIndex';",
+            PRE_LEXICAL_RERANK_COLLECTION_NAME, keyspaceName, PRE_LEXICAL_RERANK_COLLECTION_NAME),
+        String.format(
+            "CREATE CUSTOM INDEX IF NOT EXISTS %s_exists_keys ON \"%s\".\"%s\" (values(exist_keys)) USING 'StorageAttachedIndex';",
+            PRE_LEXICAL_RERANK_COLLECTION_NAME, keyspaceName, PRE_LEXICAL_RERANK_COLLECTION_NAME),
+        String.format(
+            "CREATE CUSTOM INDEX IF NOT EXISTS %s_query_bool_values ON \"%s\".\"%s\" (entries(query_bool_values)) USING 'StorageAttachedIndex';",
+            PRE_LEXICAL_RERANK_COLLECTION_NAME, keyspaceName, PRE_LEXICAL_RERANK_COLLECTION_NAME),
+        String.format(
+            "CREATE CUSTOM INDEX IF NOT EXISTS %s_query_dbl_values ON \"%s\".\"%s\" (entries(query_dbl_values)) USING 'StorageAttachedIndex';",
+            PRE_LEXICAL_RERANK_COLLECTION_NAME, keyspaceName, PRE_LEXICAL_RERANK_COLLECTION_NAME),
+        String.format(
+            "CREATE CUSTOM INDEX IF NOT EXISTS %s_query_null_values ON \"%s\".\"%s\" (values(query_null_values)) USING 'StorageAttachedIndex';",
+            PRE_LEXICAL_RERANK_COLLECTION_NAME, keyspaceName, PRE_LEXICAL_RERANK_COLLECTION_NAME),
+        String.format(
+            "CREATE CUSTOM INDEX IF NOT EXISTS %s_query_text_values ON \"%s\".\"%s\" (entries(query_text_values)) USING 'StorageAttachedIndex';",
+            PRE_LEXICAL_RERANK_COLLECTION_NAME, keyspaceName, PRE_LEXICAL_RERANK_COLLECTION_NAME),
+        String.format(
+            "CREATE CUSTOM INDEX IF NOT EXISTS %s_query_timestamp_values ON \"%s\".\"%s\" (entries(query_timestamp_values)) USING 'StorageAttachedIndex';",
+            PRE_LEXICAL_RERANK_COLLECTION_NAME, keyspaceName, PRE_LEXICAL_RERANK_COLLECTION_NAME)
+      };
+      for (String indexCql : createIndexCqls) {
+        assertThat(executeCqlStatement(SimpleStatement.newInstance(indexCql))).isTrue();
+      }
+    }
+
+    @Test
+    @Order(2)
+    public final void createCollectionWithoutLexicalRerankAgain() {
+      givenHeadersPostJsonThenOkNoErrors(
+                  """
+                      {
+                          "createCollection": {
+                              "name": "%s"
+                          }
+                      }
+              """
+                  .formatted(PRE_LEXICAL_RERANK_COLLECTION_NAME))
+          .body("$", responseIsStatusOnly())
+          .body("status.ok", is(1));
```

Comment:
Same here, listCollections (or whatever command there is), to check Collection exists before call and after
(could otherwise pass if Collection was not yet created if tests ran in wrong order, or CQL table creation had failed.